projects/syntheticBabylon/index.md

```diff
@@ -0,0 +1,103 @@
+kaum länger als ein Fingernagel.
+3. Verbinde die Enden der Senkrechten jeweils mit den beiden
+Punkten durch gerade Linien.
+4. Zeichne eine beliebige Anzahl von Senkrechten zwischen diese
+Schrägen.
+5. Verbinde die entstandenen oberen Berührungspunkte auf den
+Schrägen mit den noch nicht verbundenen Punkt.
+6. Setze diese Technik fort bis ein Gefühl von Stadtraum entsteht
+
+Vielen Dank!
+
+The results turned out to be rough and did not fulfil the visual expectations.
+Therefore in the following steps the algorithm has continuously been adjusted to the impressions of natural and organic structures.
+
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT1/1.png)
+
+
+### Iteration 2:
+
+Arbeite die folgenden Punkte nacheinander ab.
+Nutze dafür ein blankes Blatt Papier und ein Bleistift (wenn
+möglich).
+
+1. Zeichne zwei Punkte die waagerecht zueinander stehen, und
+nicht weiter entfernt sind als ein Finger lang ist.
+2. Zeichne unterhalb – zwischen diesen Punkten eine Senkrechte,
+kaum länger als ein Fingernagel.
+3. Verbinde die Enden der Senkrechten jeweils mit den beiden
+Punkten durch gerade Linien.
+4. Zeichne eine beliebige Anzahl von Senkrechten zwischen diese
+Schrägen.
+5. Verbinde die entstandenen oberen Berührungspunkte auf den
+Schrägen mit den noch nicht verbundenen Punkt.
+6. Die Sonne geht im Osten auf.
+7. Bist du mit dem Ergebnis zufrieden? Was könnte man
+ergänzen?
+
+Vielen Dank!
+
+This text was handed to 10 people.
+
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT2/1.png)
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT2/2.png)
+
+For the final iteration the algorithm was adjusted to achieve a richer picture.
+
+### Iteration 3/4/5:
+
+Arbeite die folgenden Punkte nacheinander ab.
+Nutze dafür ein blankes Blatt Papier und ein Bleistift (wenn
+möglich).
+
+1. Zeichne zwei Punkte die waagerecht zueinander stehen, und
+nicht weiter entfernt sind als ein Finger lang ist.
+2. Zeichne unterhalb – zwischen diesen Punkten eine Senkrechte,
+kaum länger als ein Fingernagel.
+3. Verbinde die Enden der Senkrechten jeweils mit den beiden
+Punkten durch gerade Linien.
+4. Zeichne eine beliebige Anzahl von Senkrechten zwischen diese
+Schrägen.
+5. Verbinde die entstandenen oberen Berührungspunkte auf den
+Schrägen jeweils mit dem gegenüberliegenden Punkt aus (1.)
+der noch nicht verbunden ist.
+6. Setze diese Technik fort, indem du weitere Schrägen,
+Senkrechten und vielleicht sogar Punkte zeichnest. Kannst du
+eine schwebende Stadt erschaffen?
+
+Vielen Dank!
+
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT3/1.png)
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT3/2.png)
+![](https://github.com/t0dorakis/syntheticBabylon/blob/master/assets/visuals/IT3/3.png)
+
+
+The picture underneath is my favorite one and it is closest to what I tried to achieve.
```

Comment:
Can you write a bit more conclusion? Not only I like the last one. Did you get anything out of this exercise or was it plain stupid to you?